yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -978,15 +978,14 @@ func fetchUnsupportedQueryConstructs() ([]utils.UnsupportedQueryConstruct, error
 	return result, nil
 }
 
-func fetchColumnsWithUnsupportedDataTypes() ([]utils.TableColumnsDataTypes, []utils.TableColumnsDataTypes, error) {
-	var unsupportedDataTypes []utils.TableColumnsDataTypes
-	var unsupportedDataTypesForLiveMigration []utils.TableColumnsDataTypes
+func fetchColumnsWithUnsupportedDataTypes() ([]utils.TableColumnsDataTypes, []utils.TableColumnsDataTypes, []utils.TableColumnsDataTypes, error) {
```

Comment:
Overall comment about this function
- It is getting a bit complicated to read and understand the code
- Can we introduce more helper function(proper naming), under which we hide the usage of utils functions etc...
- nit: add switch case in place of if-else for checking what bucket of unsupportedDatatypes it should fall to - `unsupportedDataTypes, unsupportedDataTypesForLiveMigration, unsupportedDataTypesForLiveMigration`